src/screens/User/components/podcastItem.tsx

```diff
@@ -1,16 +1,33 @@
 import React from 'react'
 import { View, Image, TouchableOpacity } from 'react-native'
 import { Heading, HeadingType, PlainText } from 'src/components'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { NavigationScreen } from 'src/common/enums'
+import { RootStackParamList } from 'src/common/types'
 import { Podcast } from 'src/common/types'
 import styles from './styles'
 
+type EpisodeScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  NavigationScreen.MY_PROFILE
+>
+
 type Props = {
   podcast: Podcast
+  navigation: EpisodeScreenNavigationProp
 }
 
-const Item: React.FC<Props> = ({ podcast }) => {
+const Item: React.FC<Props> = ({ podcast, navigation }) => {
+  const handleBackToPodcast = () => {
```

Comment:
```suggestion
  const handleBack = () => {
```

Reply:
Thats my bad, thats not BackToPodcast, what it does thats redirect us to podcast,